model/index.js

```diff
@@ -64,13 +60,41 @@ module.exports = yeoman.Base.extend({
 
   addNullDataSourceItem: actions.addNullDataSourceItem,
 
+  setBluemixDatasourceState: function() {
+    if (this.options.bluemix) {
+      var configPath = path.resolve(process.cwd(), '.bluemix',
```

Comment:
Let's use `this.destinationPath('.bluemix/datasources-config.json')` instead. See http://yeoman.io/authoring/file-system.html.